src/crypto/equihash.h

```diff
@@ -108,6 +108,8 @@ class TruncatedStepRow : public StepRow<WIDTH>
     eh_trunc* GetTruncatedIndices(size_t len, size_t lenIndices) const;
 };
 
+inline constexpr const size_t max(const size_t A, const size_t B) { return A > B ? A : B; }
```

Comment:
Not in C++ 11, because `std::max` is only `const`, not `constexpr`. C++ 14 fixes this.